OpenRA.Mods.Common/Traits/BotModules/SupportPowerBotModule.cs

```diff
@@ -110,7 +110,7 @@ void IBotTick.BotTick(IBot bot)
 					// Valid target found, delay by a few ticks to avoid rescanning before power fires via order
 					AIUtils.BotDebug("{0} found new target location {1} for support power {2}.", player.PlayerName, attackLocation, sp.Info.OrderName);
 					waitingPowers[sp] += 10;
-					bot.QueueOrder(new Order(sp.Key, supportPowerManager.Self, Target.FromCell(world, attackLocation.Value), false) { SuppressVisualFeedback = true });
+					bot.QueueOrder(new Order(sp.Key, supportPowerManager.Self, Target.FromCell(world, attackLocation.Value), false) { SuppressVisualFeedback = true, ExtraData = (uint)world.LocalRandom.Next(1024) }); // HACK: ExtraData for Airstrike Power
```

Comment:
Use `uint.MaxValue` instead of explicitly picking a direction, and the comment should reference `SelectDirectionalTarget` which is what defines this behaviour.

Reply:
>Use uint.MaxValue

I don't get how to use this on random direction here

>comment should reference SelectDirectionalTarget 
Got it.

Reply:
SelectDirectionalTarget uses `uint.MaxValue` to indicate that the player did not pick a direction, and that the support power should choose one following its default rules (in this case: randomly).

Reply:
I see, will do that

Reply:
fixed